src/ray/gcs/store_client/test/store_client_test_base.h

```diff
@@ -70,6 +70,8 @@ class StoreClientTestBase : public ::testing::Test {
       RAY_CHECK_OK(store_client_->AsyncPut(table_name_, elem.first.Binary(),
                                            elem.second.SerializeAsString(),
                                            put_calllback));
+      RAY_CHECK_OK(store_client_->AsyncPut(table_name_, elem.first.Binary(),
```

Comment:
Add a comment about why we need the 2nd call with `nullptr` in addition to the same args?